runtime/frequency/src/lib.rs

```diff
@@ -157,7 +164,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("frequency"),
 	impl_name: create_runtime_str!("frequency"),
 	authoring_version: 1,
-	spec_version: 3,
+	spec_version: 4,
```

Comment:
🙇

Reply:
.9.32 will have number 5 so once that is merged out overall spec_version rococo-mainnet will be synchronizing to 5